.circleci/build_local_docker.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+set -e
+
+thisdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+docker build -t localtest -f- "$thisdir"<<EOF
+FROM registry.opensuse.org/devel/openqa/ci/containers/base:latest
+
+RUN sudo zypper ar -f -p 80 https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.1/openSUSE_Leap_15.1 openQA
+RUN sudo zypper ar -f -p 81 http://download.opensuse.org/repositories/devel:openQA/openSUSE_Leap_15.1 devel
```

Comment:
The hardcoded version of 15.1 is now duplicated multiple times. Can you extract that?

Reply:
easy to search & replace

Reply:
That depends on what `ci/containers/base:latest` was build on. We probably should move it to git at some point, but in any case I don't think that abstraction here will help in current state. I was even about to propose `build_local_docker_lp151.sh` and `build_local_docker_tw.sh` , which wouldn't rely only on raw OS (no dependency on CI image); I can do it later if somebody sees benefit from it.